validator/src/main/java/com/google/daq/mqtt/sequencer/SequenceBase.java

```diff
@@ -392,6 +393,9 @@ private Config readGeneratedConfig() {
    */
   @Before
   public void setUp() {
+    // Old messages can sometimes take a while to clear out, so need some delay for stability.
+    safeSleep(CLEAN_START_DELAY_MS);
```

Comment:
Can we add a TODO to minimize this time or change the strategy?

Reply:
Done.